es6.js

```diff
@@ -0,0 +1,59 @@
+class BuilderES6 {
+    constructor(integer) {
+        this.integer = integer || 0;
+    }
+}
+
+class IntBuilder extends BuilderES6 {
+    plus(...n) { 
```

Comment:
Could you please explain your reasoning behind not using constructor here in `IntBuilder` class ?